test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -244,11 +273,43 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
      *
      * @return none
      */
-    function resetPermissions() {
+    function resetPermissions()
+    {
         $this->DB->run(
             "INSERT IGNORE INTO user_perm_rel SELECT 999990, PermID FROM permissions"
         );
     }
 
+    /**
+     * Helper function to determine if a menu item is prensent
+     * under a given category.
+     *
+     * Note: By giving and empty string in any parameter will act as a joker.
+     *   ex: assertMenuItemPresent('', 'Configuration') will return true if
+     *       one or more menu item 'Configuration' exist under any category.
+     *   ex: assertMenuItemPresent('Admin', '') will return true is any
+     *       category 'Admin' has at least one  menu item.
+     *
+     * @param string $category The parent Label of the menu item
+     * @param string $menu     The label of the menu item
+     *
+     * @return boolean true    If the menu item exists under the category.
+     */
+    function assertMenuItemPresent($category, $menu)
+    {
+        $configMenu = $this->webDriver->findElements(
+            WebDriverBy::xPath(
+                "
+                //ul[@class='nav navbar-nav']
+                //a[contains(text(), '$category')]
+                /..
+                /ul[@class='dropdown-menu']
+                //a[contains(text(), '$menu')]
+                "
+            )
+        );
+        return count($configMenu) > 0;
+    }
+
```

Comment:
Should there be an assertMenuItemNotPresent that checks count == 0?

(That would make it easier to do permission tests on the menu items..)